app/src/main/java/com/monkeyteam/chimpagne/model/location/Weather.kt

```diff
@@ -0,0 +1,14 @@
+package com.monkeyteam.chimpagne.model.location
+
+import java.time.LocalDate
+
+data class Weather(
+    val location: Location? = Location(),
+    val date: LocalDate = LocalDate.now(),
+    val weatherDescription: String = "",
+    val weatherIcon: String = "",
+    val temperatureLow: Double = 0.0,
+    val temperatureHigh: Double = 0.0,
+    val maxWindSpeed: Double = 0.0,
+    val windDirection: String = ""
```

Comment:
storing the direction as a String is a bit odd, ideally it should be degrees so a float would make more sense

Reply:
Changed to your liking sir, good comment.